packages/eslint-plugin/README.md

```diff
@@ -24,7 +24,7 @@ Simply add this plugin in your `.eslintrc` file to use the add the plugin. The p
 
 ```json
 plugins: [
-    "@blueprintjs/blueprint"
+    "@blueprintjs/eslint-plugin"
```

Comment:
```suggestion
    "@blueprintjs"
```

actually this can just be `@blueprintjs`
<img width="674" alt="image" src="https://user-images.githubusercontent.com/723999/88189317-ee69c280-cc06-11ea-82e6-7c5522bd95a3.png">